fix packing issue

D3vil0p3r opened the pull request:
Sorry again @skelsec . As in https://github.com/skelsec/kerberoast/pull/9 , also minikerberos has the same issue of the `tests` folder in the root python folder. I propose the same solution used for kerberoast for fixing this packing issue.

Thanks

D3vil0p3r commented:
Hey @skelsec did you have the chance to give a look to this PR?

skelsec commented:
Hello,
thank you for the PR and headsup about this issue. I got the same issue reported on my other packages as well, and I fixed (hopefully) all of them. As this issue was affecting a bigger number of packages I fixed it via a script. 
Therefore I'll not be accepting your PR as it is now considered fixed as of version 0.4.1
Can you please verify?

D3vil0p3r commented:
> Hello, thank you for the PR and headsup about this issue. I got the same issue reported on my other packages as well, and I fixed (hopefully) all of them. As this issue was affecting a bigger number of packages I fixed it via a script. Therefore I'll not be accepting your PR as it is now considered fixed as of version 0.4.1 Can you please verify?

Verified. It is fixed. I close this PR. Thank you man.
